app/views/alert_mailer/daily_alert.text.haml

```diff
@@ -15,7 +14,13 @@
   \---
   \
 \
-Visit #{notify_link('https://teaching-vacancies.service.gov.uk', t('app.title'))} - the free service for schools in England to list teaching roles and for jobseekers to find them.
+\# #This job alert
```

Comment:
I guess this text 'This job alert' (and the below 'Visit') should be in the en.yml.